doc/api/esm.md

```diff
@@ -1178,23 +1195,30 @@ export async function resolve(specifier, context, defaultResolve) {
     };
   }
   if (anotherCondition) {
-    // When calling the defaultResolve, the arguments can be modified. In this
-    // case it's adding another value for matching conditional exports.
-    return defaultResolve(specifier, {
+    // When calling to the parent loader, the arguments can be modified. In
+    // this case it's adding another value for matching conditional exports.
+    return parentLoader.resolve(specifier, parentURL, {
       ...context,
       conditions: [...context.conditions, 'another-condition'],
     });
   }
   // Defer to Node.js for all other specifiers.
-  return defaultResolve(specifier, context, defaultResolve);
+  return null;
 }
 ```
 
-#### <code>getFormat</code> hook
+#### <code>getFormat(url, context)</code> hook
```

Comment:
See previous note about leaving the signatures out of the headings. Also, this signature doesn't match the API.